services/outputhost/cgcache.go

```diff
@@ -500,9 +502,55 @@ func (cgCache *consumerGroupCache) refreshCgCache(ctx thrift.Context) error {
 		}
 	}
 
+	newDLQMerging := nSingleCGExtents > 0
+	if cgCache.dlqMerging != newDLQMerging {
+		cgCache.dlqMerging = newDLQMerging
+		if newDLQMerging {
+			cgCache.logger.WithField(`nSingleCGExtents`, nSingleCGExtents).Info("Merging DLQ Extent(s)")
```

Comment:
The feature is called 'DLQ merge'. Someone looking for code related to merging will want to look at this.